test/integration/json-api/v1/base-format/update/index.js

```diff
@@ -270,6 +269,8 @@ describe('updatingResources', function() {
     describe('200Ok', function() {
 
       // TODO: No idea why this suddenly started returning 204 instead of 200
```

Comment:
I'd like to keep this one for now. Even if json-api relaxes the requirements, they're still RFC 7231 requirements, and I'd like to test for them. The 200 and 409 tests passed at some point.